src/unix/tcp.c

```diff
@@ -115,6 +115,10 @@ int uv__tcp_bind(uv_tcp_t* tcp,
                    IPV6_V6ONLY,
                    &on,
                    sizeof on) == -1) {
+#if defined(__MVS__)
+      if (errno == EOPNOTSUPP)
+        return -EINVAL;
```

Comment:
Test "tcp_bind6_error_inval" expects EINVAL when binding a socket to an already bound loopback address. 
However, on z/OS the error instead of EINVAL is EOPNOTSUPP.
Do you think I should instead change the test case to expect EOPNOTSUPP on z/OS?